session/bootstrap.go

```diff
@@ -424,6 +424,10 @@ const (
 	CreateAdvisoryLocks = `CREATE TABLE IF NOT EXISTS mysql.advisory_locks (
 		lock_name VARCHAR(64) NOT NULL PRIMARY KEY
 	);`
+	// CreateMDLView is a view about metadata locks.
+	CreateMDLView = `CREATE OR REPLACE VIEW mysql.tidb_mdl_view as (
```

Comment:
Maybe we can add some tests / usage examples for this view. Of course, you can add in the next PR.

Reply:
The next PR tests this view.